### DISCUSSION TITLE
libgccjit 11.3 on homebrew change `.so` to `.dylib` makes some breaking changes.

### DISCUSSION THREAD

**Discussion by ccqpein** [Homebrew/discussions#3222] [Everyday usage]
I just upgrade `libgccjit` this morning and one of my build scripts fails after that. Then I notice the `libgccjit.so` file in `/usr/local/opt/libgccjit/lib/gcc/` changed to `libgccjit.0.dylib`. 

Is there a way I can build libgccjit through homebrew and also bring `.so` back?

**Comment by Bo98** [comment]
`.dylib` is the standard library extension on macOS, not `.so`.

What you are seeing is a consequence of upstream fixing a bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100613 (not marked as fixed, but it was in 11.3.0). The dylib extension is correct behaviour and you will see all macOS distributions of libgccjit use it from now on, Homebrew or not.
